File: lab3 - non-native - flutter/domain/Recepie.dart
class Recepie {

  int id;
  String title;
  String type;
  List<String> ingredients;
  String description;

  Recepie(this.id, this.title, this.type, this.ingredients, this.description);
  Recepie.withoutId(this.title, this.type, this.ingredients, this.description);
  
  String ingredientsToString() {
    String result = "";
    for(String ing in ingredients){
      result += " "+ ing;
    }
    return result;
  }
  
}

File: lab5 - non-native with DB & REST server services/server/recepie server/core/src/main/java/coreBase/Entities/Recepie.java
package coreBase.Entities;

import lombok.*;

import javax.persistence.Entity;

/**
 * @author Rares.
 */
@Entity
@NoArgsConstructor
@AllArgsConstructor
@Data
@EqualsAndHashCode(callSuper = true)
@ToString(callSuper = true)
@Builder
public class Recepie extends BaseEntity<Integer> {

    private String title;
    private String type;
    private String description;
    private String ingredients;

}
